File: Volume 5 (1400–1499)/1409.py
# timus problem no: 1409
# problem link: http://acm.timus.ru/problem.aspx?space=1&num=1409

H, L = raw_input().split()
H = int(H)
L = int(L)
total = (H + L) - 1
H = total - H
L = total - L

print "%d %d \n" % (H, L)

File: Volume 5 (1400–1499)/1409.cpp
/*
# timus problem no: 1409
# problem link: http://acm.timus.ru/problem.aspx?space=1&num=1409
*/

#include <stdio.h>

int main()
{
    int H, L, total;
    scanf("%d %d", &H, &L);
    total = (H + L) - 1;
    printf("%d %d\n", total - H, total - L);

    return 0;
}
